cmd/cluster-storage-operator/main.go

```diff
@@ -22,6 +22,7 @@ func main() {
 	pflag.CommandLine.AddGoFlagSet(flag.CommandLine)
 
 	logs.InitLogs()
+	logs.AddFlags(pflag.CommandLine)
```

Comment:
Please check this PR for an example of how this should look like: https://github.com/openshift/cluster-kube-controller-manager-operator/pull/581/commits/c339a947622142be4e6e0df7799cb2a1445e1eb2

Reply:
Oh that's way better, thank you, updated :)